app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str):
+    command = command.split()
+    command_values = {
+        "mode": command[0],
+        "file": command[1],
+        "file_move": command[2]
+    }
```

Comment:
what for you use dictionary here? You can just create 3 variables and use them without complication